test/jdk/tools/launcher/Jexec.java

```diff
@@ -23,14 +23,25 @@
 
 /*
  * @test
- * @bug 8175000
+ * @bug 8175000 8314491
  * @summary test jexec
+ * @requires os.family == "linux"
  * @build TestHelper
  * @run main Jexec
  */
 
+import java.beans.Transient;
+import java.io.BufferedReader;
 import java.io.File;
 import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.HashMap;
```

Comment:
Check if the imports are really used.  I can't imagine why java.beans.Transient would be needed, ditto UUID, etc.